qiita_db/archive.py

```diff
@@ -77,6 +77,19 @@ def insert_from_artifact(cls, artifact, features):
 
             cls._inserting_main_steps(ms, features)
 
+    @classmethod
```

Comment:
I think this should be an instance method, since it should be applied to the archive.